.gitpod.yml

```diff
@@ -29,6 +29,10 @@ tasks:
     command: |
       ./dev/preview/download-and-merge-harvester-kubeconfig.sh
       exit 0
+  - name: Add VM SSH keys
+    command: |
+      ./dev/preview/install-vm-ssh-keys.sh
+      exit 0
```

Comment:
What do you think of doing the whole SSH connection in one of the terminal tabs and keeping it running?
Maybe just doing a quick check to verify that the workspace has a related VM, then close this tab if it hasn't

Reply:
That is basically a good idea and would also add some comfort. I am personally a fan of doing it myself. @mads-hartmann What do you thing?

Reply:
I'm hoping that SSH access will only be needed for debugging issues with the VM, so if everything works smoothly the SSH connection shouldn't be needed most of the time - most likely developers will primarily use kubectl to get the logs etc. for their applications which we're trying to make convenient in https://github.com/gitpod-io/ops/issues/644

For for now, let's keep it as it, where developers have to run a script whenever they need SSH access ☺️

Reply:
I believe that just creating a terminal for the SSH connection already open wouldn't hurt as well 😅 

No problems doing that later though